packages/blocks/src/surface-block/managers/edit-session.ts

```diff
@@ -190,9 +190,12 @@ export class EditSessionStorage {
   };
 
   constructor(private _service: SurfaceService) {
-    const props = sessionStorage.getItem(SESSION_PROP_KEY);
-    if (props) {
-      this._lastProps = LastPropsSchema.parse(JSON.parse(props));
+    try {
+      this._lastProps = LastPropsSchema.parse(
+        JSON.parse(sessionStorage.getItem(SESSION_PROP_KEY) ?? '')
+      );
+    } catch {
+      /* empty */
```

Comment:
You should NOT swallow error in this way. Log error message or throw it explicitly.